src/parse.y

```diff
@@ -56,6 +56,9 @@
 %destructor { free($$); } QUOTED_STRING
 %destructor { free($$); } VERSION_NO
 
+%destructor { free($$); } mls_component mls_level mls_range sl_item
```

Comment:
Isn't an sl_item just a string_list with one node?

Reply:
I think its just a string (no `struct string_list`)
https://github.com/TresysTechnology/selint/blob/9221bd945e4fdc0035c063f08778d13b13febcd5/src/parse.y#L353-L363

Reply:
You're right.  I see now why I was confused.  I was getting over a cold on Tuesday morning, so I blame that.